CriticalMass/UIWindow+Appearance.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  UIWindow+Appearance.swift
+//  CriticalMaps
+//
+//  Created by Malte Bünz on 15.04.19.
+//  Copyright © 2019 Pokus Labs. All rights reserved.
+//
+
+import UIKit
+
+extension NSNotification.Name {
+    static let CMAppearanceWillRefreshWindow = NSNotification.Name(rawValue: "CMAppearanceWillRefreshWindowNotification")
+    static let CMAppearanceDidRefreshWindow = NSNotification.Name(rawValue: "CMAppearanceDidRefreshWindowNotification")
+}
+
+extension UIWindow {
+    @nonobjc private func _refreshAppearance() {
```

Comment:
Why do we have to remove and add all views in a Window?

Reply:
Otherwise changing the theme does not affect the window immediately. And to do that we have to build the view hierarchy again

Reply:
To be honest this seems a bit scary. bu it seems like everyone is doing it. We should also add a comment here.

Reply:
I had the same feeling :D I'll add it